conda/recipes/cudf/meta.yaml

```diff
@@ -41,6 +41,7 @@ requirements:
     - {{ pin_compatible('rmm', max_pin='x.x') }}
     - fsspec>=0.6.0
     - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
+    - nvtx >=0.2.1
```

Comment:
Do we want to constrain this to `nvtx >=0.2.1,<0.3.0a0` or are we comfortable we won't be introducing breaking changes? cc @shwina

Reply:
I don't think we need to constrain it. There's no plan for a `0.3.0` any time soon